tests/test_flows.py

```diff
@@ -132,6 +133,47 @@ def __capture(self, test_index, test_name, bridge, duration=None):
                     tshark_args+['-a', 'duration:900'], shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         except subprocess.CalledProcessError as e:
             self.err(f'test {self.run_tests} failed, output: {e.output}')
+    def check_tshark(self, target_thread_name=None, end=time.time(), start=None, file_output=False):
+        '''
+        ### Description
+        Looks into the tshark captured data, updating the global capture data
+        and returning the captured data from `start` to `end`
+
+        ### Parameters
+        `target_thread_name=None` (str) dictionary key (in self.tshark_instances) of the thread running tshark, if None, the method
+        defaults to self.running (i.e. the current running test)
+
+        `end=time.time()` (datetime) upper bound for captured packets timestamp
+
+        `start=None` (datetime) lower bound for captured packets timestamp
+
+        `file_output=False` flag to determine if the captured data would be 
+        written to a file
+
+        ### Returns
+
+        a collection of captured `ieee1905` packets
```

Comment:
*Very* good documentation, we can use this as an example (for CODING_STYLE.md) of how to write docstrings.

Just one thing though: the first line should be a summary:

```
    '''Get tshark captured data between `start` and `end`.

    ### Description
    ...
```

Reply:
Can I suggest we stick to a single style of docstring?
What we've been using in the recent python scripts we added ([capi.py](https://github.com/prplfoundation/prplMesh/blob/master/tests/capi.py) for example) is based on Numpy's docstring: https://numpydoc.readthedocs.io/en/latest/format.html

I don't want to impose this one over another, but I think we should definitely choose one and try to stick to it.

Reply:
Here is more or less what it would look like using Numpy's style:
```suggestion
        """Get tshark captured data between `start` and `end`.
        
        Looks into the tshark captured data, updating the global capture data
        and returning the captured data from `start` to `end`.

        Parameters
        ----------
        target_thread_name: str, optional
            Dictionary key (in self.tshark_instances) of the thread running tshark (default is None, which means the current running test will be used).
        end: datetime
            Upper bound for captured packets timestamp.
        start: datetime, optional
            Lower bound for captured packets timestamp (default is None).
        file_output: bool, optional
            Flag to determine if the captured data would be written to a file (default is False).

        Returns
        -------
        List[replace_with_the_classname_of_temp_capture]
            A collection of captured `ieee1905` packets.
```